python_modules/dagster/dagster/_daemon/controller.py

```diff
@@ -74,6 +74,7 @@ def create_daemon_grpc_server_registry(
 ) -> GrpcServerRegistry:
     return GrpcServerRegistry(
         instance_ref=instance.get_ref(),
+        server_command=GrpcServerCommand.CODE_SERVER_START,
```

Comment:
i think maybe this path should still use the old command? The place where I this path will be used is people running the daemon directly, without dagster dev (but not running their own code server). In that case I don't think we get any benefit from using the new command?

There is maybe an argument for just migrating everything over to the new code server command though, just to keep things consistent. I think it is possible that it uses a bit more memory since there are now two python processes instead of one, that is the main downside I can think of.

Reply:
Let's leave it out of this PR then, and we can change it in another if we so desire.

Reply:
Let's leave it out of this PR then, and we can change it in another if we so desire.